bin/generate-shell-css.js

```diff
@@ -0,0 +1,25 @@
+var http = require( 'http' );
+var critical = require( 'critical' );
+
+// Load the HTML page from the dev server. This will fail if the dev server is
```

Comment:
@DanReyLop Success! 😄 

I got `shell.css` generated at build time (yay!) by manually running `jade` with a stubbed-out set of variables.

Could you sanity-check `server/pages.index.js`? I had to move away from using `include` in the jade template, so instead I'm injecting the inline JS/CSS through the `context` object. But since I'm getting the content asynchronously (`fs.readFile`), I had to make `getDefaultContext` an async function. I think it's all good, but an extra set of eyes would be nice 😄 

Thanks for calling this out. I think this is a much nicer solution, in general.